docker-compose/readme.md

```diff
@@ -279,5 +279,25 @@ NOTES:
   * For the CBC related setup see these files for more details:
     * `Jenkinsfiles/Jenkinsfile.cbc-run-integration-tests` - Jenkinsfile for running the tests in a CBC project/job. 
     * `Jenkinsfiles/cbc-run-integration-tests.yaml` - Kubernetes docker container specification.  These settings will also need to be updated when there are CBC image naming changes.
-   
 
+## Developing and Running Selenium tests in Local Development
+You can run selenium tests by following below steps:
+
+  1. Make sure there is no blue button server and its dependent services running
+  2. Go to the base directory of the local repo and run:
+
+     use MSLSX (default)   
+     ```
+     ./docker-compose/run_integration_tests_local_keybase.sh
```

Comment:
Tried running this locally and ran into some issues:
```
ip-10-232-52-234:bluebutton-web-server nickbragdon$ ./docker-compose/run_integration_tests_local_keybase.sh mslsx
run_integration_tests_local_keybase.sh: 
run_integration_tests_local_keybase.sh: RUNNING SCRIPT: ./docker-compose/run_integration_tests_local_keybase.sh
run_integration_tests_local_keybase.sh: 

COMMAND USAGE HELP
------------------

  Use one of the following command line options for the type of test to run:

    local  = Run using the docker-compose web local developer setup + local bfd (QUICK test)

    local-debug  = Same as 'local' with tests run waiting on port 6789 to be attached

    dc  = Run using the docker-compose web local developer setup (QUICK test)

    dc-debug  = Same as 'dc' with tests run waiting on port 5678 to be attached

    cbc = Run using the docker CBC (Cloud Bees Core) containter image (FULL test)
```
That was the result when trying the command with mslsx. I got the same result when trying it with no inputs.
Then we I tried it with input based on the response and got the following:
```
ip-10-232-52-234:bluebutton-web-server nickbragdon$ ./docker-compose/run_integration_tests_local_keybase.sh dc
run_integration_tests_local_keybase.sh: 
run_integration_tests_local_keybase.sh: RUNNING SCRIPT: ./docker-compose/run_integration_tests_local_keybase.sh
run_integration_tests_local_keybase.sh: 
run_integration_tests_local_keybase.sh:  - Setting Keybase location based on SYSTEM type: Darwin
run_integration_tests_local_keybase.sh: 
run_integration_tests_local_keybase.sh:  - Sourcing ENV secrets from: /keybase/team/bb20/infrastructure/creds/ENV_secrets_for_local_integration_tests.env
run_integration_tests_local_keybase.sh: 
run_integration_tests_local_keybase.sh: 
run_integration_tests_local_keybase.sh: ERROR: The ENV secrets could NOT be found at: /keybase/team/bb20/infrastructure/creds/ENV_secrets_for_local_integration_tests.env
run_integration_tests_local_keybase.sh: 
```
I've mounted keybase locally but I'm not sure if it works the same on Mac

Reply:
@nbragdon thanks for trying, seems the integration test script has been run : ./docker-compose/run_integration_tests_local_keybase.sh mslsx
run_integration_tests_local_keybase.sh: 
run_integration_tests_local_keybase.sh: RUNNING SCRIPT: ./docker-compose/run_integration_tests_local_keybase.sh
run_integration_tests_local_keybase.sh: 

the selenium one is: run_selenium_tests_local_keybase.sh

Reply:
Hmmm I guess I'm confused - I thought the doc was outlining how to run the selenium test?
`You can run selenium tests by following below steps:`
We should instead be using: `run_selenium_tests_local_keybase.sh`?